config/routes.rb

```diff
@@ -3,4 +3,24 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources(
+    :scientists,
+    only: %i[show]
+  )
+
+  resources(
+    :scientist_experiments,
+    only: %i[destroy]
+  )
```

Comment:
It looks like you still have an issue with nested routing. I think this is something you should focus on wrapping your head around going into the intermission.